osCam/templates/user/login.html

```diff
@@ -0,0 +1,23 @@
+<html>
+  <head>
+    {% include "core/bootstrap.html" %}
+    {% load static %}
+  </head>
+  <body>
+    {% include "core/navigation.html" %}
+    <div class="container">
+      <div class = "jumbotron">
+        <h1>Login</h1>
+        <form action="/login/" method="POST">{% csrf_token %}
+          <table>
+            {{ login_form.as_table }}
+            <tr><td>
+              <input type="submit" class="btn btn-primary" value = "Login"/>
+            </td></tr>
+            <!-- <tr><td>Click <a href="/join">here</a> to join.</td></tr> -->
```

Comment:
clean up unused code and make meaningul comments when doing so

Reply:
believe ive addressed this. there could be more useful comments but i think i've at least gotten rid of useless ones